src/ui/ButtonLink.tsx

```diff
@@ -1,26 +1,69 @@
 'use client';
 
-import { chakra } from '@chakra-ui/react';
+import { Text } from '@/ui/Text';
+import { Flex, Icon, chakra } from '@chakra-ui/react';
+import { ArrowRight } from '@phosphor-icons/react';
 import { ReactNode, forwardRef } from 'react';
 
-import { Button, ButtonProps } from './Button';
 import { Link, LinkProps } from './Link';
 import { NextLink } from './NextLink';
-import { buttonRecipe } from './theme/recipes/ButtonRecipe';
-
-const ButtonLinkBase = forwardRef<
-  HTMLAnchorElement,
```

Comment:
Perhaps I should keep this and rename the new button link component RedesignButtonLink. Then slowly we'll get rid of the old ButtonLink. Thoughts?

Reply:
I think it's easier to stick with 'redesign' prefix when we start to remove the old code for the sake of consistency, but not a big deal IMO if you're already done with this part.